spot_driver/spot_driver/spot_ros2.py

```diff
@@ -2919,6 +2929,43 @@ def handle_set_gripper_camera_parameters(
 
         return response
 
+    def handle_arm_velocity_command(self, arm_velocity_command: ArmVelocityCommandRequest) -> None:
+        if self.spot_wrapper is None:
+            return
+
+        cylindrical_velocity = arm_command_pb2.ArmVelocityCommand.CylindricalVelocity()
+        cylindrical_velocity.linear_velocity.r = arm_velocity_command.command.cylindrical_velocity.linear_velocity.r
+        cylindrical_velocity.linear_velocity.theta = (
+            arm_velocity_command.command.cylindrical_velocity.linear_velocity.theta
+        )
+        cylindrical_velocity.linear_velocity.z = arm_velocity_command.command.cylindrical_velocity.linear_velocity.z
+
+        angular_velocity = geometry_pb2.Vec3(
+            x=arm_velocity_command.angular_velocity_of_hand_rt_odom_in_hand.x,
+            y=arm_velocity_command.angular_velocity_of_hand_rt_odom_in_hand.y,
+            z=arm_velocity_command.angular_velocity_of_hand_rt_odom_in_hand.z,
+        )
+
+        cartesian_velocity = arm_command_pb2.ArmVelocityCommand.CartesianVelocity()
+        cartesian_velocity.velocity_in_frame_name.x = (
+            arm_velocity_command.command.cartesian_velocity.velocity_in_frame_name.x
+        )
+        cartesian_velocity.velocity_in_frame_name.y = (
+            arm_velocity_command.command.cartesian_velocity.velocity_in_frame_name.y
+        )
+        cartesian_velocity.velocity_in_frame_name.z = (
+            arm_velocity_command.command.cartesian_velocity.velocity_in_frame_name.z
+        )
+        cartesian_velocity.frame_name = arm_velocity_command.command.cartesian_velocity.frame_name
+
+        proto_command = arm_command_pb2.ArmVelocityCommand.Request(
+            cylindrical_velocity=cylindrical_velocity,
+            angular_velocity_of_hand_rt_odom_in_hand=angular_velocity,
+            cartesian_velocity=cartesian_velocity,
+        )
+
+        return self.spot_wrapper.spot_arm.handle_arm_velocity(proto_command)
```

Comment:
```suggestion
        return self.spot_wrapper.spot_arm.handle_arm_velocity(proto_command, self.cmd_duration)
```
If we want this to mimic the `cmd_vel` topic, we can use the same duration field as it.